src/Tests/Microsoft.NET.TestFramework/Commands/ComposeStoreCommand.cs

```diff
@@ -20,24 +21,23 @@ public ComposeStoreCommand(ITestOutputHelper log, string projectPath, string rel
         }
 
         public override DirectoryInfo GetOutputDirectory(string targetFramework = "netcoreapp1.0", string configuration = "Debug", string runtimeIdentifier = "")
-        {
-            string output = Path.Combine(ProjectRootPath, "bin", BuildRelativeOutputPath(targetFramework, configuration, runtimeIdentifier));
-            return new DirectoryInfo(output);
-        }
-
-        public string GetPublishedAppPath(string appName)
-        {
-            return Path.Combine(GetOutputDirectory().FullName, $"{appName}.dll");
-        }
-
-        private string BuildRelativeOutputPath(string targetFramework, string configuration, string runtimeIdentifier)
         {
             if (runtimeIdentifier.Length == 0)
             {
                 runtimeIdentifier = RuntimeInformation.RuntimeIdentifier;
             }
             string arch = runtimeIdentifier.Substring(runtimeIdentifier.LastIndexOf("-") + 1);
-            return Path.Combine(configuration, arch, targetFramework, PublishSubfolderName);
+            string relativeToBaseOutputPath = Path.Combine(configuration, arch, targetFramework, runtimeIdentifier, PublishSubfolderName);
+            return base.GetBaseOutputDirectory().Sub(relativeToBaseOutputPath);
+
+            // TODO: SDK-Style projects must not contain arch in the platform. Replace the above logic with the following once it's fixed.
```

Comment:
TODO

Reply:
We already use `ProcessorArchitecture` (via `Platform`) in the `RuntimeIdentifier`, so, there's no need to duplicate it in the paths, at least in the tests!

But if we want to match the default paths from Common targets, I could modify the methods to accept `arch` as an optional parameter.

Reply:
@wli3

Reply:
What do you mean by breaking output paths? Isn't this internal change?